study-builder/fdahpStudyDesigner/src/main/webapp/WEB-INF/view/errorPage.jsp

```diff
@@ -0,0 +1,158 @@
+<%@ page language="java" contentType="text/html; charset=UTF-8" pageEncoding="UTF-8" %>
```

Comment:
Copyright header

Reply:
In the existing `JSP `files we have not added copyright header